src/App.tsx

```diff
@@ -26,70 +27,138 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType) {
+    visibleGoods.sort((good1, good2) => {
+      if (sortType === SortType.LENGTH) {
+        return good1.length - good2.length;
+      }
+
+      if (sortType === SortType.ALPHABET) {
+        return good1.localeCompare(good2);
+      }
+
+      return 0;
+    });
+  }
```

Comment:
No need to sort if sortType is NONE
```suggestion
  if (sortType !== SortType.NONE) {
    visibleGoods.sort((good1, good2) => {
      if (sortType === SortType.LENGTH) {
        return good1.length - good2.length;
      }

      return good1.localeCompare(good2);
    });
  }
```